### PR TITLE
EVSS: Get base urls and sample user from env

### PR DESCRIPTION
**Pull request by carlmjohnson**
This will let use test our dev integration by setting up different YAML files for the servers.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ before_script:
   - export MHV_SM_HOST='https://mock-prescriptions-api.herokuapp.com'
   - export MHV_SM_APP_TOKEN='a-fake-app-token'
   - export MHV_SM_USER_ID='10616687'
+  - export EVSS_CLAIMS_BASE_URL='http://csraciapp6.evss.srarad.com:7003/wss-claims-services-web-3.1/rest'
+  - export EVSS_DOCUMENTS_BASE_URL='http://csraciapp6.evss.srarad.com:7003/wss-document-services-web-3.1/rest/'
+  - >
+    export EVSS_SAMPLE_CLAIMANT_USER='{"first_name": "Jane","last_name": "Doe","issue_instant": "2015-04-17T14:52:48Z","edipi": "1105051936","participant_id": "123456789"}'
```
Comment (austin-martinez) [inline .travis.yml]: Any reason this can't all fit on L27?

File: app/models/user.rb [hunk 2]
```diff
@@ -24,12 +24,6 @@ class User < RedisStore
...
 
   def self.sample_claimant
-    User.new(
-      first_name: 'Jane',
-      last_name: 'Doe',
-      issue_instant: '2015-04-17T14:52:48Z',
-      edipi: '1105051936',
-      participant_id: '123456789'
-    )
+    User.new JSON.load ENV['EVSS_SAMPLE_CLAIMANT_USER']
```
Comment (austin-martinez) [inline app/models/user.rb]: I prefer at least inner parens when nesting function calls like this
`User.new JSON.load(ENV['EVSS_SAMPLE_CLAIMANT_USER'])`

File: .travis.yml (hunk 1 above)
Comment (carlmjohnson) [inline .travis.yml]: I was too lazy to escape all the quotation marks, so I looked up the syntax for quoting a whole line. I agree that it's too obscure for other readers. I'll go ahead and escape it.

File: app/models/user.rb (hunk 2 above)
Comment (carlmjohnson) [inline app/models/user.rb]: 👌

**Pull request merged**